dockers/actions-plan-preview/main.go

```diff
@@ -56,6 +56,32 @@ func main() {
 	}
 	log.Printf("Successfully parsed GitHub event\n\tbase-branch %s\n\thead-branch %s\n\thead-commit %s\n", event.BaseBranch, event.HeadBranch, event.HeadCommit)
 
+	doComment := func(body string) {
+		comment, err := sendComment(
+			ctx,
+			ghClient,
+			event.Owner,
+			event.Repo,
+			event.PRNumber,
+			body,
+		)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		log.Printf("Successfully commented plan-preview result on pull request\n%s\n", *comment.HTMLURL)
+	}
+
+	if event.PRClosed {
+		doComment(failureBadgeURL + "It was unable to run plan-preview for a closed pull request.")
```

Comment:
```suggestion
		doComment(failureBadgeURL + "Unable to run plan-preview for a closed pull request.")
```
nits (same with the others), but it's up to you 👍

Reply:
👍